src/vs/workbench/parts/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -111,6 +113,34 @@ export class ExtensionsViewletViewsContribution implements IWorkbenchContributio
 		};
 	}
 
+	private createEnabledExtensionsListViewDescriptor(): IViewDescriptor {
+		return {
+			id: 'extensions.enabledList',
+			name: localize('enabledExtensions', "Enabled"),
+			container: VIEW_CONTAINER,
+			ctor: EnabledExtensionsView,
+			when: ContextKeyExpr.and(ContextKeyExpr.not('searchExtensions')),
+			weight: 30,
+			canToggleVisibility: true,
```

Comment:
We should add `canToggleVisibility: true` to the "Installed" view as well. If someone chooses to have just the Enabled section or Enabled and Disabled section separately, then it doesnt make sense to not let them hide the "Installed" section.

cc @sandy081

Reply:
I think seeing only Enabled extensions might be what many users want. Showing again installed extensions might add noise. So users can configure to see enabled extensions and hide installed. 

Also how about configuring the default views to show on open? We can have a good defaults (Enabled and Recommended) and hide the installed view by default? Users can customise from the context menu to what to see by default?

Reply:
@sandy081 

> We can have a good defaults (Enabled and Recommended) and hide the installed view by default? Users can customise from the context menu to what to see by default?

That sounds good. I'll add that in a separate commit